target/linux/ramips/dts/mt7628an_asus_rt-ac1200.dtsi

```diff
@@ -73,13 +73,22 @@
 			};
 
 			factory: partition@40000 {
+				compatible = "nvmem-cells";
 				label = "factory";
 				reg = <0x40000 0x10000>;
-				read-only;
-
-				compatible = "nvmem-cells";
 				#address-cells = <1>;
 				#size-cells = <1>;
+				read-only;
+
+				eeprom_factory_0: eeprom@0 {
+					reg = <0x0 0x400>;
+				};
+
+				/* v2 eeprom size is 0x4da8, 'reg'
+				   will be overridden in v2 dts. */
```

Comment:
Think it's better to add this in the v2 dts instead of here in the dtsi. Also for multiline comments use 

```
/*
 *
 */
```